src/field_theory/polynomial_galois_group.lean

```diff
@@ -12,22 +12,29 @@ import ring_theory.eisenstein_criterion
 /-!
 # Galois Groups of Polynomials
 
-In this file we introduce the Galois group of a polynomial, defined as
-the automorphism group of the splitting field.
+In this file, we introduce the Galois group of a polynomial `p` over a field `F`,
+defined as the automorphism group of its splitting field. We also provide
+some results about some extension `E` above `p.splitting_field`, and some specific
+results about the Galois groups of ℚ-polynomials with specific numbers of non-real roots.
 
 ## Main definitions
 
-- `gal p`: the Galois group of a polynomial p.
-- `restrict p E`: the restriction homomorphism `(E ≃ₐ[F] E) → gal p`.
-- `gal_action p E`: the action of `gal p` on the roots of `p` in `E`.
+- `polynomial.gal p`: the Galois group of a polynomial p.
+- `polynomial.gal.restrict p E`: the restriction homomorphism `(E ≃ₐ[F] E) → gal p`.
+- `polynomial.gal.gal_action p E`: the action of `gal p` on the roots of `p` in `E`.
 
 ## Main results
 
-- `restrict_smul`: `restrict p E` is compatible with `gal_action p E`.
-- `gal_action_hom_injective`: the action of `gal p` on the roots of `p` in `E` is faithful.
-- `restrict_prod_inj`: `gal (p * q)` embeds as a subgroup of `gal p × gal q`.
-- `gal_action_hom_bijective_of_prime_degree`: An irreducible polynomial of prime degree
-  with two non-real roots has full Galois group
+- `polynomial.gal.restrict_smul`: `restrict p E` is compatible with `gal_action p E`.
+- `polynomial.gal.gal_action_hom_injective`: `gal p` acting on the roots of `p` in `E` is faithful.
+- `polynomial.gal.restrict_prod_injective`: `gal (p * q)` embeds as a subgroup of `gal p × gal q`.
+- `polynomial.gal.card_of_separable`: For a separable polynomial, its Galois group has cardinality
+equal to the dimension of its splitting field over `F`.
+- `polynomial.gal.card_complex_roots_eq_card_real_add_card_not_gal_inv`: The number of complex roots
+equals the number of real roots plus the number of roots not fixed by complex conjugation
+(i.e. with some imaginary component).
```

Comment:
In my view this isn't a main result. A result like this is good to have, but I don't think that `(gal_action_hom p ℂ (restrict p ℂ (complex.conj_alg_equiv.restrict_scalars ℚ))).support.card` is the best way to state the number of non-real roots. Instead, I view this lemma as just being an auxiliary lemma for `gal_action_hom_bijective_of_prime_degree`. The name change is good though.

Reply:
I feel like it's the sort of result people take for granted in "real maths", though, so I wanted it to be easier to find. I'll put it in "other results", maybe?